P150a VMs failing profiler regression tests

Card type: p150a
FW ver: 80.17.0.0
KMD ver: 1.32

https://github.com/tenstorrent/tt-metal/actions/runs/14452242941/job/40532965302
```
FAILED tests/tt_metal/tools/profiler/test_device_profiler.py::test_timestamped_events - AssertionError: Wrong erisc event count
assert 1600 in [0, 200, 1000]
FAILED tests/tt_metal/tools/profiler/test_device_profiler.py::test_sub_device_profiler - AssertionError: Log process script crashed with exit code 256
assert 256 == 0
```

`test_timestamped_events` was failing because the expected BH results were using WH configuration - num of active eth cores needed to be updated here.
Not able to repro any failure on `test_sub_device_profiler`. 

I see:
```
FAILED tests/tt_metal/tools/profiler/test_device_profiler.py::test_dispatch_cores - AssertionError: Wrong dispatch zone count, read 992 which is not within 20 cycle counts of any of the limits [153, 246, 365, 1035, 1548, 1634, 2447]
```
@mo-tenstorrent is this test flaky on BH and do the limits need to be updated?

--- a/tests/tt_metal/tools/profiler/test_device_profiler.py
+++ b/tests/tt_metal/tools/profiler/test_device_profiler.py
@@ -364,7 +364,7 @@ def test_timestamped_events():
     ZONE_COUNT = 100
     WH_ERISC_COUNTS = [0, 1, 5]
     WH_TENSIX_COUNTS = [72, 64, 56]
-    BH_ERISC_COUNTS = [0, 1, 5]
+    BH_ERISC_COUNTS = [0, 1, 6, 8]
     BH_TENSIX_COUNTS = [130, 120, 110]
 
     WH_COMBO_COUNTS = []

--- a/tt_metal/hw/firmware/src/active_erisc.cc
+++ b/tt_metal/hw/firmware/src/active_erisc.cc
@@ -113,7 +113,7 @@ int main() {
             // Only include this iteration in the device profile if the launch message is valid. This is because all
             // workers get a go signal regardless of whether they're running a kernel or not. We don't want to profile
             // "invalid" iterations.
-            DeviceZoneScopedMainN("ACTIVE-ERISC-FW");
+            DeviceZoneScopedMainN("ERISC-FW");
             uint32_t launch_msg_rd_ptr = mailboxes->launch_msg_rd_ptr;
             launch_msg_t* launch_msg_address = &(mailboxes->launch[launch_msg_rd_ptr]);
 

--- a/tt_metal/hw/firmware/src/active_erisck.cc
+++ b/tt_metal/hw/firmware/src/active_erisck.cc
@@ -31,7 +31,7 @@ void kernel_launch(uint32_t kernel_base_addr) {
     noc_local_state_init(NOC_INDEX);
 
     {
-        DeviceZoneScopedMainChildN("ACTIVE-ERISC-KERNEL");
+        DeviceZoneScopedMainChildN("ERISC-KERNEL");
         WAYPOINT("K");
         kernel_main();
         WAYPOINT("KD");

